AY2122S1-CS2103T-T11-2/tp
# Implement Slideshow Functionality

### Description


Implemented slideshow functionality

TODO:

### Type of change


### How to test


- Run the `slideshow` command
  - Use `next` and `previous` to go to the next and previous flashcard respectively
    - Try going to the previous slide when on the first slide to make sure this cannot be done
    - Try going to the next slide when on the last slide to make sure this cannot be done
  - Use the `answer` command to answer a flashcard. The answer should be displayed on the slide
    - Try answering a flashcard that has already been answered to make sure this cannot be done
    - When navigating to next and previous slides, answered slides should still have their answers showing and cannot be answered again
  - Try typing other commands like `add`, `edit`, `delete`, etc. to make sure they cannot be executed in slideshow mode
- Try using the `next`, `previous`, and `answer` commands when out of slideshow mode to make sure this cannot be done

### Checklist



- [ ] I have updated the Developer Guide
